pkg/pulumiyaml/run.go

```diff
@@ -1064,7 +1064,7 @@ func (e *programEvaluator) registerConfig(intm configNode) (interface{}, bool) {
 	if errors.Is(err, config.ErrMissingVar) && defaultValue != nil {
 		v = defaultValue
 	} else if err != nil {
-		return e.errorf(intmKey, err.Error())
+		return e.errorf(intmKey, "%s", err.Error())
```

Comment:
This isn't the same? If Error has %'s in it doesn't that now trip the formatter that values weren't passed in resulting in MISSING outputs.

Reply:
indeed but the old way is a non constant string and golint complains

Reply:
Isn't it the other way around, previously if `err.Error()` has a "%s", it would result in `%!s(MISSING)`, but now it's just a literal `%s`.

E.g. 

```
	fmt.Printf("%s", "test has a %s inside\n")
	fmt.Printf("test has a %s inside")
```

prints

```
test has a %s inside
test has a %!s(MISSING) inside
```

So I think the first one/the way Brandon changed it looks to me more like what we want.

Reply:
Oh it was too early in the morning, I read the diff backwards somehow. Yes this is now correct, the old one was wrong for the reasons given.